protobuf/buf.gen.ts-connect.yaml

```diff
@@ -1,4 +1,4 @@
-version: v1
+version: v1.0.0 # x-release-please-version 
```

Comment:
This repo is yet to have a release, this version implies the first released version won't be v1.0.0